javascript.js

```diff
@@ -8,16 +8,30 @@ const pitcherConfig = require('./pitcher.config')
 
 module.exports = {
   extends: ['eslint:recommended'],
-  plugins: ['sort-imports-es6-autofix'],
-  rules: {
-    'sort-imports-es6-autofix/sort-imports-es6': [
-      'error',
-      {
-        ignoreCase: false,
-        ignoreMemberSort: false,
-        memberSyntaxSortOrder: ['all', 'single', 'multiple', 'none'],
+  plugins: ['import', 'node'],
+  settings: {
+    'import/resolver': {
+      webpack: {
+        config: 'node_modules/@vue/cli-service/webpack.config.js',
       },
+    },
+  },
+  rules: {
+    'import/order': [
+      'error', {
+        groups: ['builtin', 'external', 'internal', 'parent', 'sibling', 'index', 'object', 'type'],
+        pathGroups: [
+          {
+            pattern: '@/**',
+            group: 'external',
```

Comment:
asking because I don't really know: 
shouldn't this be internal?

Reply:
This is internal, yeah. But we put this AFTER the external group :D